bigquery_etl/public_data/publish_json.py

```diff
@@ -55,7 +55,9 @@ def __init__(
         self.storage_client = storage_client
         self.temp_table = None
         self.date = None
-        self.stage_gcs_path = self.gcs_path + "stage/json/"
+        self.stage_gcs_path = self.gcs_path + "stage/json/" + "".join(
+            random.choice(string.ascii_lowercase) for i in range(12)
+        )
```

Comment:
With multiple export tasks running in parallel and writing to the same stage directory, once one job is finished it will also delete staged files from all other jobs that are still running. Once they are complete and try to delete their staged files (which are already deleted) an error occurs